extensions/registries/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strictNullChecks": true,
```

Comment:
why this one it not `strict` ? too much changes ?

Reply:
Ops, i forgot it. Updated, thanks!! 👍